lldb/include/lldb/Target/Target.h

```diff
@@ -201,6 +201,8 @@ class TargetProperties : public Properties {
 
   bool GetSwiftUseTasksPlugin() const;
 
+  bool GetSwiftCacheTaskPtrLocation() const;
```

Comment:
```suggestion
  bool GetSwiftCacheTaskPointerLocation() const;
```
unless that's the canonical name for it

Reply:
It needs to follow the name of the setting in the td file, so we would spell "pointer" there too, which makes sense to help with the `apropos` command